src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double area;
```

Comment:
the same, lets remove this field in all classes

Reply:
not removed